inc/search.class.php

```diff
@@ -5305,6 +5310,25 @@ static function giveItem($itemtype, $ID, array $data, $num, $meta = 0,
                   return $CFG_GLPI['languages'][$data[$num][0]['name']][0];
                }
                return __('Default value');
+            case 'progressbar':
+               if (!isset($progressbar_data)) {
+                  $bar_color = 'green';
+                  $progressbar_data = [
+                     'percent'      => $data[$num][0]['name'],
+                     'percent_text' => $data[$num][0]['name'],
+                     'color'        => $bar_color
+                  ];
+               }
+
+               $out = "{$progressbar_data['text']}<div class='center' style='background-color: #ffffff; width: 100%;
```

Comment:
Looks like `$progressbar_data['text']` can be undefined, if it used default values defined above.

Reply:
Fixed in 896b76f9e4989c8c828e5eb95bbb94534e0a0ad9